website/docs/cli/commands/fmt.mdx

```diff
@@ -50,7 +50,7 @@ Usage: `terraform fmt [options] [TARGET]`
 By default, `fmt` scans the current directory for configuration files. If
 you provide a directory for the `target` argument, then `fmt` will scan that directory instead. If you provide a file, then `fmt` will process just that file. If you provide a single dash (`-`), then `fmt` will read from standard input (STDIN).
 
-The command-line flags are all optional. The list of available flags are:
+The command-line flags are all optional. The available flags are:
```

Comment:
Hi @sebhcp Thank you for your efforts on this PR.

I was wondering we could try to reword each of the introductory sentences to say "The following flags are available:"? Our style guidelines say we should word sentences that introduce lists in this manner to make them appear more polished and professional.

Reply:
Absolutely, I have submitted the corresponding commit.